### ISSUE TITLE
(Bug) Collections' description editor inserts image under the page, not inside the editor

### ISSUE THREAD

**Issue by CosmoMyzrailGorynych** [itchio/itch.io#1194]
**Severity: medium.** The non-critical site function is broken, but there are no (known to me) workarounds.

**TL;DR:** Can't insert images into collection's description.
**Expected behavior:** One should be able to add images to the collection's description.

**Steps to reproduce:**

1. [Open My Library](https://itch.io/my-collections) (aka My Collections)
2. Click on one of your collections' title to open its page. You will open a page with an URL `https://itch.io/c/{CollectionID}/{collection-slug}`
3. Click "Edit collection"
4. Upload or insert an existing image. Note that the editor is hidden while you select an image. (Is the editor temporarily removed from the root `document`?)
5. Observe that the editor hadn't opened again and that the image is imported at the bottom of the page.

Videos, tables, and links work. Only image upload & insertion is broken.

**Video showing the issue**

https://user-images.githubusercontent.com/3174357/118422523-7f220980-b717-11eb-8065-6fe9559cb33c.mp4
